tests/neg/i11994.check

```diff
@@ -1,26 +1,12 @@
--- [E008] Not Found Error: tests/neg/i11994.scala:2:28 -----------------------------------------------------------------
-2 |implicit def foo[T <: Tuple.meow]: Unit = ??? // error
```

Comment:
I think `implicit def foo[T <: Tuple.meow]: Unit` should still produce an error?